Userland/Libraries/LibVideo/PlaybackManager.cpp

```diff
@@ -34,7 +34,7 @@ DecoderErrorOr<NonnullOwnPtr<PlaybackManager>> PlaybackManager::from_file(String
 DecoderErrorOr<NonnullOwnPtr<PlaybackManager>> PlaybackManager::from_mapped_file(NonnullRefPtr<Core::MappedFile> mapped_file)
 {
     auto demuxer = TRY(Matroska::MatroskaDemuxer::from_mapped_file(move(mapped_file)));
-    return create_with_demuxer(move(demuxer));
+    return create(move(demuxer));
```

Comment:
Isn't create_with_demuxer removed in the previous commit? :thonk: This hunk should be moved back one.

Reply:
Ah, oversight in the rebase, I'll fix this.